source/install/self-managed-changelog.md

```diff
@@ -5,12 +5,154 @@
+
+### Compatibility
+ - Updated the minimum version of MacOS to 11+.
+
+### Highlights
+
+#### Calls
+ - 
+
+#### Boards
+ - 
+ 
+#### Playbooks
+ - 
+
+#### Message Priority and Acknowledgments (Professional Plan)
+ - Added message priority labels to the Threads view.
+ - Added support for enterprise users to request acknowledgements on posts and to acknowledge posts.
+
+#### Global Drafts
+ - Added a centralized page for draft messages.
+
+### Improvements
+
+#### User Interface (UI)
+ - Implemented progressive image loading in the webapp.
+ - When the "Custom Brand Text" is left blank with custom branding enabled, the default text is now hidden.
+ - The **Mark as Unread** option was added to the ``…`` menu for channels in the left-hand side sidebar. Pressing Alt while selecting a channel on the left-hand side now also marks the last post in the channel as unread.
+ - Channel members are now able to remove themselves from a channel via the right-hand side channel members list.
+ - Added a Monthly versus Annual pricing toggle to the pricing modal.
+ - Removed video check to allow the browser to decide what video types it can play.
+ - Added a tooltip to the right-hand side files filter icon.
+ - The number of users that can be added to a user group at once was increased to 256.
+ - Keyboard and focus handling was improved in profile popovers and at-mentions.
+ - Updated prepackaged version of plugins affected by React 17 upgrade.
+
+#### Administration
+ - If an Admin encounters an invitation error “SMTP is not configured in System Console", a link to the SMTP configuration within the **System Console** is now included in the error message.
+ - Crashing jobs now sets the job status to "failed".
+ - **Total Active Users** was renamed to **Total Activated Users** in **System Console > Reporting > Site Statistics**.
+ - Optimized ``ThreadStore.MarkAllAsUnreadByTeam``.
+ - SQL migrations for PostgreSQL will now filter by the current schema name when checking for information from the ``information_schema.columns`` view. This does not affect anything because usually there's only one installation in a given database, but this gives flexibility to users to store multiple Mattermost instances under a single database.
+ - **My Insights** and OpenId Connect were added to the Free plan.
+ - Team scheme APIs are now allowed to be administered with a Professional plan.
+ - All integration limits and usage limit components were removed.
+ - A global banner as well as a notice banner are displayed to admins on the **Invite** modal and on **System Console > Site Statistics > Total Active Users** page when the workspace exceeds the maximum number of users allowed. If the number of actual users exceeds the number of paid users by less than 5%, the banner is dismissible. If the number of actual users exceeds the number of paid users by more than 10%, the banner is non-dismissible until the license seat count has been updated.
```

Comment:
```suggestion
 - A global banner as well as a notice banner are displayed to admins on the **Invite** modal and on **System Console > Site Statistics > Total Activated Users** page when the workspace exceeds the maximum number of users allowed. If the number of actual users exceeds the number of paid users by less than 5%, the banner is dismissible. If the number of actual users exceeds the number of paid users by more than 10%, the banner is non-dismissible until the license seat count has been updated.
```

Reply:
@laneycs - Can you please confirm that "Total Active Users" should now read as "Total Activated Users" for both the Site Statistics page and the Team Statistics page in the System Console?